src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySinkConfig.java

```diff
@@ -344,9 +344,12 @@ public void validate(@Nullable Schema inputSchema, @Nullable Schema outputSchema
         String name = field.getName();
         // BigQuery column names only allow alphanumeric characters and _
```

Comment:
remove old doc reference

Reply:
In that case write one comment on top mentioning keeping these comments as it is in preview, will be removed after GA